blogposts/inside-rust/01-announcement.md

```diff
@@ -0,0 +1,166 @@
+# Announcing the first FFI-unwind project design meeting
+
+-- Kyle Strand and Niko Matsakis on behalf of the FFI-unwind project group --
+
+The FFI-unwind project group, announced in [this RFC][rfc-announcement], is
+working to extend the language to support unwinding that crosses FFI
+boundaries.
+
+We have reached our first technical decision point, on a question we have been
+discussing internally for quite a while. This blog post lays out the arguments
+on each side of the issue and invites the Rust community to join us at the
+[upcoming meeting](meeting-link) to help finalize our decision, which will be
+formalized and published as our first language-change RFC. This RFC will
+propose an "MVP" specification for well-defined cross-language unwinding.
+
+The meeting will be on Monday the 24th at 17:00 UTC, via Zoom:
+
+> Meeting ID: 768 231 760
+> https://mozilla.zoom.us/j/768231760
```

Comment:
For some reason, I feel uncomfortable posting these details here -- can we just link to the [calendar invite](https://calendar.google.com/event?action=TEMPLATE&tmeid=MjlxdW9kdmRjdHJ1cWVtYWlhc2ZvcXIzcGFfMjAyMDAyMjRUMTcwMDAwWiByZWN1ZDRiOW84Y21jMG01cm1yMDMzcDVua0Bn&tmsrc=recud4b9o8cmc0m5rmr033p5nk%40group.calendar.google.com&scp=ALL) instead?

Reply:
I can take these out. I already included a link to an "Are We Meeting Yet" page, which has the advantage of not requiring a Google log-in to view. Should I keep that link, or replace it with the invite link?

Reply:
@nikomatsakis I'm about to merge this as-is, but Amanieu will make a PR with further changes, so once that's open you can weigh in again on which link(s) to include (Google and/or arewemeetingyet).